src/components/record-form/index.tsx

```diff
@@ -539,7 +540,7 @@ const RecordForm = ({
               { label: 'Nei', value: false },
               { label: 'Ja', value: true }
             ]}
-            onChange={handleChange}
+            onChange={() => setFieldValue('dataProtectionImpactAssessment.conducted', !values.dataProtectionImpactAssessment.conducted)}
```

Comment:
Egentlig er det vel ikke gyldig å sette input som boolean verdi, det skal være string iflg. html. Virker jo også som det er  det verdien er satt til i utgangspunket selv om man har prøvd å sette value: true/false for hver radio-felt. Jeg mener vi må lagre formen med verdier i string, men transformere verdiene om ved lesing og skriving fra/til endepunkt. Se hvordan det er gjort i api-registration.

Reply:
Konverterer nå verdiene i record-page/redux/actions.ts